pkg/oci/builder_test.go

```diff
@@ -247,19 +313,13 @@ func validateOCI(path string, t *testing.T) {
 		return files[i].Path < files[j].Path
 	})
 
-	expectedFiles := []fileInfo{
-		{Path: "/etc/pki/tls/certs/ca-certificates.crt"},
-		{Path: "/etc/ssl/certs/ca-certificates.crt"},
-		{Path: "/func", Type: fs.ModeDir},
-		{Path: "/func/README.md"},
-		{Path: "/func/f", Executable: true},
-		{Path: "/func/func.yaml"},
-		{Path: "/func/go.mod"},
-		{Path: "/func/handle.go"},
-		{Path: "/func/handle_test.go"},
-	}
-
-	if diff := cmp.Diff(expectedFiles, files); diff != "" {
+	if diff := cmp.Diff(expected, files); diff != "" {
 		t.Error("files in oci differ from expectation (-want, +got):", diff)
 	}
 }
+
+type fileInfo struct {
+	Path       string
+	Type       fs.FileMode
+	Executable bool
```

Comment:
I guess it might be a time to add `Contetnt []byte` member (like [here](https://github.com/knative/func/blob/19509e5053a1ef41c485da2bb988cb7e49dc74d7/pkg/filesystem/filesystem_test.go#L34)). For regular files it would be a file content, for symlinks it would be symlink target. Maybe we could skip content for regular files for now.

Reply:
Sounds good.  There's now a member for the header's `Linkname` member, which should verify the link was added correctly.